summit.html

```diff
@@ -109,10 +68,418 @@ <h1>Agenda</h1>
+                <details>
+                  <summary>CDF, CDEvents and Secure SW Supply Chains</summary>
+                  <div>
+                    <p>
+                      <ul>
+                        <li>What is the Continuous Delivery Foundation?</li>
+                        <li>What is CDEvents, what is its current status and how
+                          does it relate to Eiffel?</li>
+                        <li>How could events improve security in the Software
+                          Supply Chains?</li>
+                      </ul>
+                    </p>
+                    <p>
+                      <b>Speakers:</b> Fatih Degirmenci &amp; Emil Bäckmark
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Intermediate</span></td>
+              <td>9:30</td>
+            </tr>
+
+            <tr><td>Break</td><td></td><td>10:30</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Submit events securely</summary>
+                  <div>
+                    <p>
+                      A proposal for securely submitting events
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Magnus Bäck
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Advanced</span></td>
+              <td>10:45</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>CLM for HW &amp; SW Combined</summary>
+                  <div>
+                    <p>
+                      How to set the confidence level on a combination of
+                      hardware and software?
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Mattias Linnér
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Advanced</span></td>
+              <td>11:15</td>
+            </tr>
+
+            <tr><td>Lunch</td><td></td><td>12:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel Ecosystem and SDKs</summary>
+                  <div>
+                    <p>
+                      How does the landscape of the Eifel ecosystem look today?
+                      What SDKs are available?
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Eiffel TC
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>13:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Orizaba - the latest Eiffel Edition</summary>
```

Comment:
I'm a typography nerd so I can help myself but to suggest using an en dash instead of a hyphen:

```suggestion
                  <summary>Orizaba – the latest Eiffel Edition</summary>
```